test/unit/multiplication/test_multiplication_general.cpp

```diff
@@ -60,13 +166,9 @@ const std::vector<std::tuple<SizeType, SizeType, SizeType, SizeType>> sizes = {
     {8, 3, 1, 3},
 };
 
-GlobalElementSize globalTestSize(const LocalElementSize& size) {
-  return {size.rows(), size.cols()};
-}
-
 template <class T, Backend B, Device D>
-void testGeneralMultiplication(const SizeType a, const SizeType b, const T alpha, const T beta,
-                               const SizeType m, const SizeType mb) {
+void testGeneralSubMultiplication(const SizeType a, const SizeType b, const T alpha, const T beta,
```

Comment:
If I understand correctly this tests the old method that will disappear.
I didn't find the tests with the "new" gemm using sub-matrices.

Reply:
Yep, you're right. Just refactored the test to include both full- and sub-matrix tests. Have a look now 👀